src/services/worker/backupUtils.ts

```
@@ -344,6 +344,8 @@ const dbRestoreSettings = async (
     localSettings.cong_settings.cong_migrated =
       settings.cong_settings.cong_migrated ?? false;
 
+    delete localSettings.cong_settings['source_material_auto_import'];
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)